terraform-aws-dataworks-pdm.tf

```
@@ -0,0 +1,69 @@
+resource "github_repository" "aws-dataworks-pdm" {
```

resources should be underscores
```suggestion
resource "github_repository" "dataworks_pdm" {
```